README.md

```diff
@@ -421,46 +404,18 @@ export async function generateEmbedTokens(userAgent, user) {
 }
 ```
 
-### Generate token frontend process
-
-When the Looker EmbedSDK is initialized for cookieless embed it requires a simple callback to call the backend process to generate new tokens.
-
-```javascript
-const generateEmbedTokensCallback = async () => {
-  const resp = await fetch('/generate-embed-tokens')
-  if (!resp.ok) {
-    console.error('generate-embed-tokens failed', { resp })
-    throw new Error(
-      `generate-embed-tokens failed: ${resp.status} ${resp.statusText}`
-    )
-  }
-  return await resp.json()
-}
-```
-
 ### Initializing the Looker SDK in the frontend
 
-Cookieless embed is initialized by calling `LookerEmbedSDK.initCookieless` passing in the Looker host value and the two callbacks described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
+Cookieless embed is initialized by calling `LookerEmbedSDK.initCookieless` passing in the Looker host value and the the urls of the backened endpoints described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
```

Comment:
```suggestion
Cookieless embed is initialized by calling `LookerEmbedSDK.initCookieless` passing in the Looker host value and the the urls of the backend endpoints described previously. Once a Looker embed IFRAME is created it will communicate with the Embed SDK running in the host application and use the callbacks appropriately.
```